src/Raven.Client/Server/DatabaseRecord.cs

```diff
@@ -50,6 +53,8 @@ public DatabaseRecord(string databaseName)
 
         public PeriodicBackupConfiguration PeriodicBackup { get; set; }
 
+        public Dictionary<string, SubscriptionRaftState> Subscriptions;
```

Comment:
See https://github.com/ravendb/ravendb/blob/v4.0/src/Sparrow/Json/JsonDeserializationBase.cs#L154 
We're pretty stuck here, so we better work with a string key dictionary, not talking about how we serialize dictionaries, so new Dictionary<int,int>(){[1]=1} will get deserialized into {"1":1}